cg/services/sequencing_qc_service/quality_checks/checks.py

```diff
@@ -46,10 +47,16 @@ def get_sequencing_quality_check_for_case(case: Case) -> Callable:
         Workflow.TAXPROFILER,
     ]
 
+    all_samples_in_case_have_reads_workflows = [
+        Workflow.NALLO,
+    ]
+
     if workflow in case_passes_workflows:
         return SequencingQCCheck.CASE_PASSES
     elif workflow in any_sample_in_case_has_reads_workflows:
         return SequencingQCCheck.ANY_SAMPLE_IN_CASE_HAS_READS
+    elif workflow in all_samples_in_case_have_reads_workflows:
+        return SequencingQCCheck.ALL_SAMPLES_IN_CASE_HAVE_READS
```

Comment:
I was meaning to give a comment here:

"""
This is starting to look a bit messy, would it be good to have a dictionary instead, so that we get the Sequencing check from the workflow?
"""

Reply:
That is a good idea, I've refactored in the last commit.